docs/src/IntroductionPage.js

```diff
@@ -0,0 +1,215 @@
+
+                <h3>JSX Syntax</h3>
+
+                <p>
+                  While each React component is really just a Javascript object,
+                  writing tree-structures that way gets tedious.
+                  React encourages the use of a syntactic-sugar called JSX,
+                   which lets you write the tree in an HTML-like syntax:
+                </p>
+
+                <div className="highlight">
+                  <pre><code className="js">{`
+var buttonGroupInstance = (
+  <ButtonGroup>
+    <DropdownButton bsStyle="success" title="Dropdown">
+      <MenuItem key="1">Dropdown link</MenuItem>
+      <MenuItem key="2">Dropdown link</MenuItem>
+    </DropdownButton>
+    <Button bsStyle="info">Middle</Button>
+    <Button bsStyle="info">Right</Button>
+  </ButtonGroup>
+);
+
+React.render(buttonGroupInstance, mountNode);
+                  `}</code></pre>
+                </div>
+
+                <p>
+                  Some people's first impression of React.js is that it seems
+                  messy to mix Javascript and HTML in this way.
+                  However, compare the code required to add
+                  a dropdown button in the example above to the
+                  <a href="http://getbootstrap.com/javascript/#dropdowns">
+                    &nbsp;Bootstrap Javascript&nbsp;
+                  </a>
+                  and
+                  <a href="http://getbootstrap.com/components/#btn-dropdowns">
+                    &nbsp;Components&nbsp;
+                  </a>
+                  documentation for creating a dropdown button.
+                  The documentation is split in two because
+                  you have to implement the component in two places
+                  in your code: first you must add the HTML/CSS elements,
+                  and then you must call some Javascript setup
+                  code to wire the component together.
+                </p>
+
+                <p>
+                  The React-Bootstrap component library tries to follow
+                  the React.js philosophy that a single piece of functionality
+                  should be defined in a single place.
+                  View the current React-Bootstrap library on the
+                  <a href="/components.html">&nbsp;components page</a>.
+                </p>
+                <p>
+                  <strong>
+                    The project is under active development - APIs will change.
+                  </strong>&nbsp;
+                  Check out the
+                  <a href="https://github.com/react-bootstrap/react-bootstrap/wiki#100-roadmap">
+                    &nbsp;1.0.0 Roadmap&nbsp;
+                  </a>
+                  and
+                  <a href="https://github.com/react-bootstrap/react-bootstrap/blob/master/CONTRIBUTING.md">
+                    &nbsp;Contributing Guidelines&nbsp;
+                  </a>
+                  to see where you can help out.
+                </p>
```

Comment:
I would rather see the contents of this paragraph tag on the home page. Something like:

![screen shot 2015-05-06 at 8 44 44 am](https://cloud.githubusercontent.com/assets/736870/7495590/2862d058-f3cc-11e4-8b17-1358d4e8bc2b.png)

Thoughts? I can PR this to your fork to include with this PR if you want or submit a PR for this myself.